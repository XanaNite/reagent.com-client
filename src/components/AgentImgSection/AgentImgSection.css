.profileBanner{
    width: 100%;
    height: 250px;
    background: rgb(238, 214, 163) url("../../images/placeholderBackground.jpg") no-repeat center;
    background-size: contain;
    text-align: center;
    color: white;
}

.profileBanner-group{
    display: inline-block;
    margin-top: 35px;
}

.profileBanner-img{
    font-size: 100px;
    margin: 25px 50px 0px 50px;
}

.profileBanner-name h2{
    font-size: 14px;
}

.overallRating{
    height: 50px;
}

@media screen and (min-width: 490px){
    .profileBanner{
        height: 400px;
        justify-content: flex-start;
        align-items: flex-end;
    }

    .profileBanner-img{
        font-size: 150px;
    }

    .profileBanner-name h2{
        margin: 0px 10px;
        font-size: 18px;
    }
}

@media screen and (min-width: 760px){
    .profileBanner{
        height: 500px;
    }

    .profileBanner-group{
        margin-top: 190px;
    }

    .profileBanner-name h2{
        font-size: 24px;
    }
}