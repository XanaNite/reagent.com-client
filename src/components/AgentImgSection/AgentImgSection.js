import React from 'react'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faUser } from '@fortawesome/free-solid-svg-icons'
import './AgentImgSection.css'

export default function AgentImgSection(props){
    return (
        <section className="agentProfile-section">
            <div className="profileBanner">
                <div className='profileBanner-img'>
                    <FontAwesomeIcon icon={faUser} />
                </div>
                <div className="profileBanner-name">
                    <h2>
                        {props.agent.first_name} {props.agent.last_name}, {props.agent.title} &#8226; {props.agent.city}, {props.agent.state}
                    </h2>
                </div>
            </div>
        </section>
    )
}