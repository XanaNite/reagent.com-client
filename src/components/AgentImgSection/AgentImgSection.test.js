import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AgentImgSection from './AgentImgSection';

describe(`AgentImgSection component`, () => {
    const props = {
        className: 'test-class-name',
        children: <p>test children</p>,
        'data-other': 'test-other-prop',
        agent: {
            first_name: "test",
            last_name: "test",
            title: "test",
            city: "test",
            state: "test"
        }
    };

    it('renders the AgentList given props', () => {
        const wrapper = shallow(<AgentImgSection {...props} />)
        expect(toJson(wrapper)).toMatchSnapshot()
    });
})