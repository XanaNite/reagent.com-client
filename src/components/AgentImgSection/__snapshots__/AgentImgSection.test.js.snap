// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentImgSection component renders the AgentList given props 1`] = `
<section
  className="agentProfile-section"
>
  <div
    className="profileBanner"
  >
    <div
      className="profileBanner-group"
    >
      <div
        className="profileBanner-img"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Object {
              "icon": Array [
                448,
                512,
                Array [],
                "f007",
                "M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z",
              ],
              "iconName": "user",
              "prefix": "fas",
            }
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          swapOpacity={false}
          symbol={false}
          title=""
          transform={null}
        />
      </div>
      <div
        className="profileBanner-name"
      >
        <h2>
          test
           
          test
          , 
          test
           â€¢ 
          test
          , 
          test
        </h2>
      </div>
    </div>
  </div>
</section>
`;
